cli/channels.go

```diff
@@ -38,36 +49,48 @@ var cmdChannels = []cobra.Command{
 				logUsage(cmd.Short)
 				return
 			}
+
 			if args[0] == "all" {
-				l, err := sdk.Channels(args[1])
+				l, err := sdk.Channels(args[1], uint64(Offset), uint64(Limit))
```

Comment:
Would it make the sense set these `Offset` and `Limit` as optional and pass them as variadic parameters or via some other means? I would like to keep SDK interface as simple as possible, not to have to explain to beginners what Offset and Limit are.

Reply:
No, these are required parameters (they are optional only in HTTP request), any user should be aware of these. Also, we should avoid package level variables (because this is easier to debug and also safer for concurrent use).